uxarray/grid/coordinates.py

```diff
@@ -102,10 +107,11 @@ def normalize_in_place(node):
     RuntimeError
         The input array doesn't have the size of 3.
     """
-    if len(node) != 3:
+    node = np.asarray(node, dtype=np.float64)
+    if node.size != 3:
         raise RuntimeError("Input array should have a length of 3: [x, y, z]")
 
-    return np.array(node) / np.linalg.norm(np.array(node), ord=2)
+    return node / np.linalg.norm(np.array(node), ord=2)
```

Comment:
That's because the actual implementation is overwritten again for some reason. This is still the old version of the implementation.

The latest implementation of the node coordinates conversion function that requires Numba fix is still unsolved.